Test/Unit/Model/PaymentTest.php

```diff
@@ -220,6 +220,27 @@ public function voidPayment_success()
         $this->currentMock->void($this->paymentMock);
     }
 
+    /**
+     * @test
+     */
+    public function voidPayment_skipHookNotification()
```

Comment:
did you mean to add this test as part of this PR?

Reply:
Nope I messed up my amend. Gone now.